platooning_control/include/pure_pursuit.hpp

```diff
@@ -1,3 +1,21 @@
+
```

Comment:
Is this a new implementation of the pure_pursuit algorithm or a copy paste of autoware's? If the latter we cannot be adding our own copyright notice to it and need to pull in autoware's instead.